<!-- saved from url=(0046)http://www.peptideatlas.org/tmp/mzML1.1.0.html -->
<!DOCTYPE html>
<html>

<head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <link rel="stylesheet" href="style/style.css"/>
</head>

<body>
<h2>Element &lt;<a name="ASAPRatio_pvalue">ASAPRatio_pvalue</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">Results of ASAPRatio pvalue analysis (adjusting ASAPRatio ratios for offset of dataset, and
            determining whether ratios are significantly different from chance values originating from 1:1 proteins
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>adj_ratio_mean</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>Ratio adjusted for dataset mean value</td>
                </tr>
                <tr>
                    <td>adj_ratio_standard_dev</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>Error adjusted for dataset mean value</td>
                </tr>
                <tr>
                    <td>heavy2light_adj_ratio_mean</td>
                    <td>xs:double</td>
                    <td>&nbsp;</td>
                    <td>Heavy2light ratio adjusted for dataset mean value</td>
                </tr>
                <tr>
                    <td>heavy2light_adj_ratio_standard_dev</td>
                    <td>xs:double</td>
                    <td>&nbsp;</td>
                    <td>Heavy2light error adjusted for dataset mean value</td>
                </tr>
                <tr>
                    <td>pvalue</td>
                    <td>xs:double</td>
                    <td>&nbsp;</td>
                    <td>pvalue (probability of ratio due by chance to 1:1 protein)</td>
                </tr>
                <tr>
                    <td>decimal_pvalue</td>
                    <td>xs:double</td>
                    <td>&nbsp;</td>
                    <td>decimal representation of pvalue</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/ASAPRatio_pvalue.png">
            <br><img width="530" src="img/ASAPRatiopvalue.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>none
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="ASAPRatio">ASAPRatio</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">Results of ASAPRatio quantitation of protein
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>ratio_mean</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>ratio_standard_dev</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>ratio_number_peptides</td>
                    <td>xs:nonNegativeInteger</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>heavy2light_ratio_mean</td>
                    <td>xs:double</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>heavy2light_ratio_standard_dev</td>
                    <td>xs:double</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>description</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>status</td>
                    <td>xs:xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>peptide_inds</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Subelement Name</th>
                    <th>min</th>
                    <th>max</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td><a href="#ASAP_Seq">ASAP_Seq</a></td>
                    <td>0</td>
                    <td>unlim</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/ASAPRatio.png">
            <br><img width="530" src="img/ASAPRatio Java.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>none
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="protein_summary">protein_summary</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>summary_xml</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Subelement Name</th>
                    <th>min</th>
                    <th>max</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td><a href="#protein_summary_header">protein_summary_header</a></td>
                    <td>1</td>
                    <td>1</td>
                    <td>Wildcard for summary info customized for a particular analysis used to infer protein
                        identifications
                    </td>
                </tr>
                <tr>
                    <td><a href="#analysis_summary">analysis_summary</a></td>
                    <td>0</td>
                    <td>unlim</td>
                    <td>time of analysis</td>
                </tr>
                <tr>
                    <td><a href="#dataset_derivation">dataset_derivation</a></td>
                    <td>1</td>
                    <td>1</td>
                    <td>Source and filtering criteria used to generate dataset</td>
                </tr>
                <tr>
                    <td><a href="#protein_group">protein_group</a></td>
                    <td>1</td>
                    <td>unlim</td>
                    <td>wildcard to be substituted with element storing analysis-specific result info</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/protein_summary.png">
            <br><img width="530" src="img/Proteinsummary.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;protein_summary xmlns="http://regis-web.systemsbiology.net/protXML" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"xsi:schemaLocation="http://sashimi.sourceforge.net/schema_revision/protXML/protXML_v6.xsd" summary_xml="/data/felipevl/testRunOfTPPPipeline/interact-UC6.prot.xml"&gt;
  &lt;protein_summary_header reference_database="/tpp.data/fasta/refseq56.Hs_plusREV.fa" residue_substitution_list="I -> L" source_files="/data/felipevl/testRunOfTPPPipeline/interact-C6.pep.xml" source_files_alt="/data/felipevl/testRunOfTPPPipeline/interact-UC6.pep.xml" min_peptide_probability="0.20" min_peptide_weight="0.50" num_predicted_correct_prots="245.5"num_input_1_spectra="0" num_input_2_spectra="2073" num_input_3_spectra="558" num_input_4_spectra="0" num_input_5_spectra="0" initial_min_peptide_prob="0.05" total_no_spectrum_ids="1565.8"sample_enzyme="trypsin"&gt;
    &lt;program_details analysis="proteinprophet" time="2015-03-31T14:39:15" version=" Insilicos_LabKey_C++ (TPP v4.7 POLAR VORTEX rev 1, Build 201410071731 (linux))"&gt;
      &lt;proteinprophet_details occam_flag="Y" groups_flag="Y" degen_flag="Y" nsp_flag="Y" fpkm_flag="N" initial_peptide_wt_iters="2" nsp_distribution_iters="4" final_peptide_wt_iters="2"run_options="XML"&gt;
        &lt;nsp_information neighboring_bin_smoothing="Y"&gt;
          &lt;nsp_distribution bin_no="0" nsp_lower_bound_incl="0.00" nsp_upper_bound_excl="0.10" pos_freq="0.095" neg_freq="0.666" pos_to_neg_ratio="0.14"/&gt;
          &lt;nsp_distribution bin_no="1" nsp_lower_bound_incl="0.10" nsp_upper_bound_excl="0.25" pos_freq="0.040" neg_freq="0.154" pos_to_neg_ratio="0.26"/&gt;
  ...
&lt;/protein_summary&gt;
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="proteinprophet_details">proteinprophet_details</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">ProteinProphet analysis details
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>occam_flag</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>whether or not occams razor was used to apportion peptides corresponding to multiple proteins
                    </td>
                </tr>
                <tr>
                    <td>groups_flag</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>whether or not related proteins are organized together in common protein groups</td>
                </tr>
                <tr>
                    <td>degen_flag</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>whether or not all proteins corresponding to each identified peptide were used in analysis</td>
                </tr>
                <tr>
                    <td>nsp_flag</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>whether or not peptide probabilities were adjusted for estimated number of sibling peptides</td>
                </tr>
                <tr>
                    <td>initial_peptide_wt_iters</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>proteinprophet number of initial iterations to compute peptide weights</td>
                </tr>
                <tr>
                    <td>nsp_distribution_iters</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>proteinprophet number of interations used to compute nsp distributions</td>
                </tr>
                <tr>
                    <td>final_peptide_wt_iters</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>proteinprophet number of final interations used to compute final peptide weights</td>
                </tr>
                <tr>
                    <td>run_options</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>special run options</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Subelement Name</th>
                    <th>min</th>
                    <th>max</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td><a href="#nsp_information">nsp_information</a></td>
                    <td>1</td>
                    <td>1</td>
                    <td>learned distributions of estimated number of sibling peptides</td>
                </tr>
                <tr>
                    <td><a href="#ni_information">ni_information</a></td>
                    <td>1</td>
                    <td>1</td>
                    <td>learned distributions of estimated number of peptide instances ("number of instances")</td>
                </tr>
                <tr>
                    <td><a href="#protein_summary_data_filter">protein_summary_data_filter</a></td>
                    <td>1</td>
                    <td>unlim</td>
                    <td>predicted error/sensitivity for filtering at minimum probability</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/proteinprophet_details.png">
            <br><img width="530" src="img/Proteinprophetdetails.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;proteinprophet_details occam_flag="Y" groups_flag="Y" degen_flag="Y" nsp_flag="Y" fpkm_flag="N" initial_peptide_wt_iters="2" nsp_distribution_iters="4" final_peptide_wt_iters="2"run_options="XML"&gt;
  &lt;nsp_information neighboring_bin_smoothing="Y"&gt;
    &lt;nsp_distribution bin_no="0" nsp_lower_bound_incl="0.00" nsp_upper_bound_excl="0.10" pos_freq="0.095" neg_freq="0.666" pos_to_neg_ratio="0.14"/&gt;
    &lt;nsp_distribution bin_no="1" nsp_lower_bound_incl="0.10" nsp_upper_bound_excl="0.25" pos_freq="0.040" neg_freq="0.154" pos_to_neg_ratio="0.26"/&gt;
    &lt;nsp_distribution bin_no="2" nsp_lower_bound_incl="0.25" nsp_upper_bound_excl="0.50" pos_freq="0.050" neg_freq="0.022" pos_to_neg_ratio="2.30"/&gt;
  ...
&lt;/proteinprophet_details&gt;
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="XPress_analysis_summary">XPress_analysis_summary</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Type:</b></td>
        <td class="zero">dx:SoftwareListType
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>min_peptide_probability</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>min_peptide_weight</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>min_protein_probability</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>reference_isotope</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/XPress_analysis_summary.png">
            <br><img width="530" src="img/XPressanalysissummary.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>none
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="ASAP_prot_analysis_summary">ASAP_prot_analysis_summary</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Type:</b></td>
        <td class="zero">dx:ScanSettingsListType
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>version</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>binary_ref_files</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>min_peptide_probability</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>min_peptide_weight</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>min_protein_probability</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>reference_isotope</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/ASAP_prot_analysis_summary.png">
            <br><img width="530" src="img/ASAPprotanalysissummary.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>none
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="ASAP_pvalue_analysis_summary">ASAP_pvalue_analysis_summary</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>asapratio_id</td>
                    <td>xs:nonNegativeInteger</td>
                    <td>&nbsp;</td>
                    <td></td>
                </tr>
                <tr>
                    <td>background_ratio_mean</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>background_ratio_stdev</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>background_fitting_error</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>analysis_distribution_file</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>full_analysis_distr_file</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>asap_prot_id</td>
                    <td>xs:nonNegativeInteger</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/ASAP_pvalue_analysis_summary.png">
            <br><img width="530" src="img/ASAPpvalueanalysissummary.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>none
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="libra_result">libra_result</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>number</td>
                    <td>xs:nonNegativeInteger</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Subelement Name</th>
                    <th>min</th>
                    <th>max</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td><a href="#intensity">intensity</a></td>
                    <td>1</td>
                    <td>unlim</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/libra_result.png">
            <br><img width="530" src="img/Libraresult.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>none
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="libra_summary">libra_summary</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>mass_tolerance</td>
                    <td>xs:float</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>centroiding_preference</td>
                    <td>xs:int</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>normalization</td>
                    <td>xs:int</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>output_type</td>
                    <td>xs:int</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>channel_code</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>min_pep_prob</td>
                    <td>xs:float</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>min_pep_wt</td>
                    <td>xs:float</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>min_prot_prob</td>
                    <td>xs:float</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Subelement Name</th>
                    <th>min</th>
                    <th>max</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td><a href="#fragment_masses">fragment_masses</a></td>
                    <td>1</td>
                    <td>unlim</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td><a href="#isotopic_contributions">isotopic_contributions</a></td>
                    <td>1</td>
                    <td>1</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/libra_summary.png">
            <br><img width="530" src="img/Librasummary.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>none
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="ASAP_Seq">ASAP_Seq</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>status</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>include</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>datanum</td>
                    <td>xs:nonNegativeInteger</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>ratio_mean</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>ratio_standard_dev</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>heavy2light_ratio_mean</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>heavy2light_ratio_standard_dev</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>weight</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>light_sequence</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Subelement Name</th>
                    <th>min</th>
                    <th>max</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td><a href="#ASAP_Peak">ASAP_Peak</a></td>
                    <td>1</td>
                    <td>unlim</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/ASAP_Seq.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>none
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="ASAP_Peak">ASAP_Peak</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>status</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>include</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>datanum</td>
                    <td>xs:nonNegativeInteger</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>ratio_mean</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>ratio_standard_dev</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>heavy2light_ratio_mean</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>heavy2light_ratio_standard_dev</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>weight</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>peptide_binary_ind</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Subelement Name</th>
                    <th>min</th>
                    <th>max</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td><a href="#ASAP_Dta">ASAP_Dta</a></td>
                    <td>1</td>
                    <td>unlim</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/ASAP_Peak.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>none
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="ASAP_Dta">ASAP_Dta</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>peptide_index</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>include</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
    </tbody>
</table>
</td>
</tr>
<tr>
    <td class="zero"><b>Example Context:</b></td>
    <td class="zero"><pre>none
</pre>
    </td>
</tr>
</tbody>
</table>
<br>

<h2>Element &lt;<a name="protein_summary_header">protein_summary_header</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>reference_database</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>full path database name</td>
                </tr>
                <tr>
                    <td>win-cyg_reference_database</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>windows full path database</td>
                </tr>
                <tr>
                    <td>residue_substitution_list</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>residues considered equivalent when comparing peptides</td>
                </tr>
                <tr>
                    <td>organism</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>sample organism (used for annotation purposes)</td>
                </tr>
                <tr>
                    <td>source_files</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>input pepXML files</td>
                </tr>
                <tr>
                    <td>source_files_alt</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>win-cyg_source_files</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>windows pepXML file names</td>
                </tr>
                <tr>
                    <td>source_file_xtn</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>file type (if not pepXML)</td>
                </tr>
                <tr>
                    <td>min_peptide_probability</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>minimum adjusted peptide probability contributing to protein probability</td>
                </tr>
                <tr>
                    <td>min_peptide_weight</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>minimum peptide weight contributing to protein probability</td>
                </tr>
                <tr>
                    <td>num_predicted_correct_prots</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>total number of predicted correct protein ids (sum of probabilities)</td>
                </tr>
                <tr>
                    <td>num_input_1_spectra</td>
                    <td>xs:integer</td>
                    <td>required</td>
                    <td>number of spectra from 1+ precursor ions</td>
                </tr>
                <tr>
                    <td>num_input_2_spectra</td>
                    <td>xs:integer</td>
                    <td>required</td>
                    <td>number of spectra from 2+ precursor ions</td>
                </tr>
                <tr>
                    <td>num_input_3_spectra</td>
                    <td>xs:integer</td>
                    <td>required</td>
                    <td>number of spectra from 3+ precursor ions</td>
                </tr>
                <tr>
                    <td>num_input_4_spectra</td>
                    <td>xs:integer</td>
                    <td>required</td>
                    <td>number of spectra from 4+ precursor ions</td>
                </tr>
                <tr>
                    <td>num_input_5_spectra</td>
                    <td>xs:integer</td>
                    <td>required</td>
                    <td>number of spectra from 5+ precursor ions</td>
                </tr>
                <tr>
                    <td>initial_min_peptide_prob</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>minimum initial peptide probability to contribute to analysis</td>
                </tr>
                <tr>
                    <td>total_no_spectrum_ids</td>
                    <td>xs:double</td>
                    <td>&nbsp;</td>
                    <td>total estimated number of correct peptide assignments in dataset</td>
                </tr>
                <tr>
                    <td>sample_enzyme</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>enzyme applied to sample prior to MS/MS</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Subelement Name</th>
                    <th>min</th>
                    <th>max</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td><a href="#program_details">program_details</a></td>
                    <td>1</td>
                    <td>1</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/protein_summary_header.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;protein_summary_header reference_database="/tpp.data/fasta/refseq56.Hs_plusREV.fa" residue_substitution_list="I -> L" source_files="/data/felipevl/testRunOfTPPPipeline/interact-C6.pep.xml" source_files_alt="/data/felipevl/testRunOfTPPPipeline/interact-UC6.pep.xml" min_peptide_probability="0.20" min_peptide_weight="0.50" num_predicted_correct_prots="245.5"num_input_1_spectra="0" num_input_2_spectra="2073" num_input_3_spectra="558" num_input_4_spectra="0" num_input_5_spectra="0" initial_min_peptide_prob="0.05" total_no_spectrum_ids="1565.8"sample_enzyme="trypsin"&gt;
  &lt;program_details analysis="proteinprophet" time="2015-03-31T14:39:15" version=" Insilicos_LabKey_C++ (TPP v4.7 POLAR VORTEX rev 1, Build 201410071731 (linux))"&gt;
    &lt;proteinprophet_details occam_flag="Y" groups_flag="Y" degen_flag="Y" nsp_flag="Y" fpkm_flag="N" initial_peptide_wt_iters="2" nsp_distribution_iters="4" final_peptide_wt_iters="2"run_options="XML"&gt;
      &lt;nsp_information neighboring_bin_smoothing="Y"&gt;
        &lt;nsp_distribution bin_no="0" nsp_lower_bound_incl="0.00" nsp_upper_bound_excl="0.10" pos_freq="0.095" neg_freq="0.666" pos_to_neg_ratio="0.14"/&gt;
        &lt;nsp_distribution bin_no="1" nsp_lower_bound_incl="0.10" nsp_upper_bound_excl="0.25" pos_freq="0.040" neg_freq="0.154" pos_to_neg_ratio="0.26"/&gt;
  ...
&lt;/protein_summary_header&gt;
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="program_details">program_details</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>analysis</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>Name of analysis used for protein identifications</td>
                </tr>
                <tr>
                    <td>time</td>
                    <td>xs:dateTime</td>
                    <td>required</td>
                    <td>Time of analysis</td>
                </tr>
                <tr>
                    <td>version</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/program_details.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;program_details analysis="proteinprophet" time="2015-03-31T14:39:15" version=" Insilicos_LabKey_C++ (TPP v4.7 POLAR VORTEX rev 1, Build 201410071731 (linux))"&gt;
  &lt;proteinprophet_details occam_flag="Y" groups_flag="Y" degen_flag="Y" nsp_flag="Y" fpkm_flag="N" initial_peptide_wt_iters="2" nsp_distribution_iters="4" final_peptide_wt_iters="2"run_options="XML"&gt;
    &lt;nsp_information neighboring_bin_smoothing="Y"&gt;
      &lt;nsp_distribution bin_no="0" nsp_lower_bound_incl="0.00" nsp_upper_bound_excl="0.10" pos_freq="0.095" neg_freq="0.666" pos_to_neg_ratio="0.14"/&gt;
  ...
&lt;/program_details&gt;
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="analysis_summary">analysis_summary</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>analysis</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>name of protein-level analysis</td>
                </tr>
                <tr>
                    <td>time</td>
                    <td>xs:dateTime</td>
                    <td>required</td>
                    <td>time of analysis</td>
                </tr>
                <tr>
                    <td>id</td>
                    <td>xs:nonNegativeInteger</td>
                    <td>required</td>
                    <td>unique id corresponding with analysis_result elements</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/analysis_summary.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>none
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="dataset_derivation">dataset_derivation</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">Source and filtering criteria used to generate dataset
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>generation_no</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>number preceding filter generations</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/dataset_derivation.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;dataset_derivation generation_no="0"&gt;
&lt;/dataset_derivation&gt;
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="protein_group">protein_group</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>group_number</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>index</td>
                </tr>
                <tr>
                    <td>pseudo_name</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>probability</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>group probability (taking contributions from all group member proteins)</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Subelement Name</th>
                    <th>min</th>
                    <th>max</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td><a href="#protein">protein</a></td>
                    <td>1</td>
                    <td>unlim</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/protein_group.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;protein_group group_number="1" probability="1.0000"&gt;
  &lt;protein protein_name="gi|100913206|ref|NP_001348.2|" n_indistinguishable_proteins="1" probability="1.0000" percent_coverage="12.4"unique_stripped_peptides="DVVQAYPEVR+ELDALDANDELTPLGR+ETPFELIEALLK+GMTLVTPLQLLLFASK+LSMSQLNEK+MGGEEAEIR+TTQVPQFILDDFIQNDR+VFDPVPVGVTK+VNLSQDLEHQLQNIIQELNLEILPPPEDPSVPVALNIGK+YGDGPRPPK+YTQVGPDHNR"group_sibling_id="a" total_number_peptides="13" pct_spectrum_ids="0.65" confidence="1.00"&gt;
    &lt;parameter name="prot_length" value="1270"/&gt;
    &lt;annotation protein_description="ATP-dependent RNA helicase A [Homo sapiens]"/&gt;
  ...
&lt;/protein_group&gt;
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="protein">protein</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>protein_name</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>database protein name</td>
                </tr>
                <tr>
                    <td>probability</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>confidence of protein id</td>
                </tr>
                <tr>
                    <td>percent_coverage</td>
                    <td>xs:double</td>
                    <td>&nbsp;</td>
                    <td>percent of protein sequence covered by corresponding peptides</td>
                </tr>
                <tr>
                    <td>n_indistinguishable_proteins</td>
                    <td>xs:integer</td>
                    <td>required</td>
                    <td>number of proteins with identical corresponding peptides</td>
                </tr>
                <tr>
                    <td>unique_stripped_peptides</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>group_sibling_id</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>protein group id</td>
                </tr>
                <tr>
                    <td>total_number_peptides</td>
                    <td>xs:integer</td>
                    <td>&nbsp;</td>
                    <td>total number of corresponding peptides that contributed to protein probability</td>
                </tr>
                <tr>
                    <td>subsuming_protein_entry</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>name of portein containing all corresponding peptides, and more</td>
                </tr>
                <tr>
                    <td>pct_spectrum_ids</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>fraction of correct dataset peptide identifications corresponding to protein</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Subelement Name</th>
                    <th>min</th>
                    <th>max</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td><a href="#parameter">parameter</a></td>
                    <td>0</td>
                    <td>unlim</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td><a href="#analysis_result">analysis_result</a></td>
                    <td>0</td>
                    <td>unlim</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td><a href="#annotation">annotation</a></td>
                    <td>1</td>
                    <td>1</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td><a href="#indistinguishable_protein">indistinguishable_protein</a></td>
                    <td>0</td>
                    <td>unlim</td>
                    <td>other protein sharing corresponding peptides</td>
                </tr>
                <tr>
                    <td><a href="#peptide">peptide</a></td>
                    <td>1</td>
                    <td>unlim</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/protein.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;protein protein_name="gi|100913206|ref|NP_001348.2|" n_indistinguishable_proteins="1" probability="1.0000" percent_coverage="12.4"unique_stripped_peptides="DVVQAYPEVR+ELDALDANDELTPLGR+ETPFELIEALLK+GMTLVTPLQLLLFASK+LSMSQLNEK+MGGEEAEIR+TTQVPQFILDDFIQNDR+VFDPVPVGVTK+VNLSQDLEHQLQNIIQELNLEILPPPEDPSVPVALNIGK+YGDGPRPPK+YTQVGPDHNR"group_sibling_id="a" total_number_peptides="13" pct_spectrum_ids="0.65" confidence="1.00"&gt;
  &lt;parameter name="prot_length" value="1270"/&gt;
  &lt;annotation protein_description="ATP-dependent RNA helicase A [Homo sapiens]"/&gt;
  ...
&lt;/protein&gt;
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="analysis_result">analysis_result</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>analysis</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>name of protein level analysis</td>
                </tr>
                <tr>
                    <td>id</td>
                    <td>xs:nonNegativeInteger</td>
                    <td>&nbsp;</td>
                    <td>nunique identifier to analysis_summary element</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/analysis_result.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>none
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="annotation">annotation</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>protein_description</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>ipi_name</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>refseq_name</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>swissprot_name</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>ensembl_name</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>trembl_name</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>locus_link_name</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>flybase</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;annotation protein_description="ATP-dependent RNA helicase A [Homo sapiens]"/&gt;
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="indistinguishable_protein">indistinguishable_protein</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">other protein sharing corresponding peptides
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>protein_name</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Subelement Name</th>
                    <th>min</th>
                    <th>max</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td><a href="#parameter">parameter</a></td>
                    <td>0</td>
                    <td>unlim</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td><a href="#annotation">annotation</a></td>
                    <td>1</td>
                    <td>1</td>
                    <td>protein description</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/indistinguishable_protein.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;indistinguishable_protein protein_name="gi|197116386|ref|NP_001127703.1|"&gt;
  &lt;annotation protein_description="5'-nucleotidase domain-containing protein 2 isoform 1 [Homo sapiens]"/&gt;
&lt;/indistinguishable_protein&gt;
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="parameter">parameter</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Type:</b></td>
        <td class="zero">nameValueType
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;parameter name="prot_length" value="520"/&gt;
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="annotation">annotation</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">protein description
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>protein_description</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>description</td>
                </tr>
                <tr>
                    <td>ipi_name</td>
                    <td>xs:IDREF</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>refseq_name</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>swissprot_name</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>ensembl_name</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>trembl_name</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>locus_link_name</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>flybase</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;annotation protein_description="5'-nucleotidase domain-containing protein 2 isoform 2 [Homo sapiens]"/&gt;
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="peptide">peptide</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>peptide_sequence</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>unmodified aa sequence</td>
                </tr>
                <tr>
                    <td>charge</td>
                    <td>xs:positiveInteger</td>
                    <td>required</td>
                    <td>precursor ion charge</td>
                </tr>
                <tr>
                    <td>initial_probability</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>precursor ion charge</td>
                </tr>
                <tr>
                    <td>nsp_adjusted_probability</td>
                    <td>xs:double</td>
                    <td>optional</td>
                    <td>after nsp adjustment
                        <
                    </td>
                </tr>
                <tr>
                    <td>ni_adjusted_probability</td>
                    <td>xs:double</td>
                    <td>optional</td>
                    <td>after ni adjustmen</td>
                </tr>
                <tr>
                    <td>exp_sibling_ion_instances</td>
                    <td>xs:double</td>
                    <td>optional</td>
                    <td>expected sibling ion instances</td>
                </tr>
                <tr>
                    <td>exp_sibling_ion_bin</td>
                    <td>xs:double</td>
                    <td>optional</td>
                    <td>expected sibling ion bin</td>
                </tr>
                <tr>
                    <td>exp_tot_instances</td>
                    <td>xs:double</td>
                    <td>optional</td>
                    <td>expected total instances</td>
                </tr>
                <tr>
                    <td>peptide_group_designator</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>link to spectra with other precursor ion charges assigned to same peptide</td>
                </tr>
                <tr>
                    <td>weight</td>
                    <td>xs:double</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>is_nondegenerate_evidence</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>confidence that peptide corresponds with this protein (rather than others in which it occurs)
                    </td>
                </tr>
                <tr>
                    <td>n_enzymatic_termini</td>
                    <td>xs:nonNegativeInteger</td>
                    <td>required</td>
                    <td>number of termini consistent with proteolytic cleavage</td>
                </tr>
                <tr>
                    <td>n_sibling_peptides</td>
                    <td>xs:double</td>
                    <td>&nbsp;</td>
                    <td>estimated number of sibling peptides (other identified peptides corresponding to same protein)
                    </td>
                </tr>
                <tr>
                    <td>n_sibling_peptides_bin</td>
                    <td>xs:integer</td>
                    <td>&nbsp;</td>
                    <td>discretized nsp value</td>
                </tr>
                <tr>
                    <td>n_instances</td>
                    <td>xs:integer</td>
                    <td>required</td>
                    <td>number of times peptide assigned to spectrum of precursor ion charge in dataset</td>
                </tr>
                <tr>
                    <td>calc_neutral_pep_mass</td>
                    <td>xs:double</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>is_contributing_evidence</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Subelement Name</th>
                    <th>min</th>
                    <th>max</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td><a href="#parameter">parameter</a></td>
                    <td>0</td>
                    <td>unlim</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td><a href="#modification_info">modification_info</a></td>
                    <td>0</td>
                    <td>unlim</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td><a href="#peptide_parent_protein">peptide_parent_protein</a></td>
                    <td>0</td>
                    <td>unlim</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td><a href="#indistinguishable_peptide">indistinguishable_peptide</a></td>
                    <td>0</td>
                    <td>unlim</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/peptide.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;peptide peptide_sequence="MGGEEAEIR" charge="2" initial_probability="0.9964" nsp_adjusted_probability="0.9994" fpkm_adjusted_probability="0.9994" weight="1.00" group_weight="1.00"is_nondegenerate_evidence="Y" n_enzymatic_termini="2" n_sibling_peptides="7.63" n_sibling_peptides_bin="6" max_fpkm="0.00" fpkm_bin="0" n_instances="1" exp_tot_instances="1.00"is_contributing_evidence="Y" calc_neutral_pep_mass="990.4437"&gt;
&lt;/peptide&gt;
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="modification_info">modification_info</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>mod_nterm_mass</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>mod_cterm_mass</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>modified_peptide</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Subelement Name</th>
                    <th>min</th>
                    <th>max</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td><a href="#mod_aminoacid_mass">mod_aminoacid_mass</a></td>
                    <td>0</td>
                    <td>unlim</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/modification_info.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;modification_info modified_peptide="Q[111]LFDVVIVQADKPSFFTDR"&gt;
  &lt;mod_aminoacid_mass position="1" mass="111.032100"/&gt;
&lt;/modification_info&gt;
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>
<h2>Element &lt;<a name="mod_aminoacid_mass">mod_aminoacid_mass</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>position</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>mass</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;mod_aminoacid_mass position="1" mass="111.032100"/&gt;
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="peptide_parent_protein">peptide_parent_protein</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>protein_name</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;peptide_parent_protein protein_name="gi|345478663|ref|NP_001230871.1|"/&gt;
&lt;peptide_parent_protein protein_name="gi|345478663|ref|NP_001230871.1|"/&gt;
&lt;peptide_parent_protein protein_name="gi|345478666|ref|NP_001230865.1|"/&gt;
&lt;peptide_parent_protein protein_name="gi|345478668|ref|NP_001230869.1|"/&gt;
&lt;peptide_parent_protein protein_name="gi|345478671|ref|NP_001230870.1|"/&gt;
&lt;peptide_parent_protein protein_name="gi|4506133|ref|NP_002758.1|"/&gt;
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="indistinguishable_peptide">indistinguishable_peptide</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>peptide_sequence</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Subelement Name</th>
                    <th>min</th>
                    <th>max</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td><a href="#modification_info">modification_info</a></td>
                    <td>0</td>
                    <td>unlim</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/indistinguishable_peptide.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>none
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>
<h2>Element &lt;<a name="modification_info">modification_info</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>mod_nterm_mass</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>mod_cterm_mass</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>modified_peptide</td>
                    <td>xs:string</td>
                    <td>&nbsp;</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>calc_neutral_pep_mass</td>
                    <td>xs:double</td>
                    <td>optional</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Subelement Name</th>
                    <th>min</th>
                    <th>max</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td><a href="#mod_aminoacid_mass">mod_aminoacid_mass</a></td>
                    <td>0</td>
                    <td>unlim</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/modification_info.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;modification_info mod_nterm_mass="43.0184" modified_peptide="n[43]MDLQAAGAQAQGAAEPSR"&gt;
    &lt;/modification_info&gt;
    </pre>
        </td>
    </tr>
    </tbody>
</table>
<br>
<h2>Element &lt;<a name="mod_aminoacid_mass">mod_aminoacid_mass</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>position</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>mass</td>
                    <td>xs:stringF</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;mod_aminoacid_mass position="5" mass="147.035400"/&gt;
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="nsp_information">nsp_information</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">learned distributions of estimated number of sibling peptides
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>neighboring_bin_smoothing</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Subelement Name</th>
                    <th>min</th>
                    <th>max</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td><a href="#nsp_distribution">nsp_distribution</a></td>
                    <td>1</td>
                    <td>unlim</td>
                    <td>histogram slice of estimated number of sibling peptides</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/nsp_information.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;nsp_information neighboring_bin_smoothing="Y"&gt;
  &lt;nsp_distribution bin_no="0" nsp_lower_bound_incl="0.00" nsp_upper_bound_excl="0.10" pos_freq="0.095" neg_freq="0.666" pos_to_neg_ratio="0.14"/&gt;
  &lt;nsp_distribution bin_no="1" nsp_lower_bound_incl="0.10" nsp_upper_bound_excl="0.25" pos_freq="0.040" neg_freq="0.154" pos_to_neg_ratio="0.26"/&gt;
  &lt;nsp_distribution bin_no="2" nsp_lower_bound_incl="0.25" nsp_upper_bound_excl="0.50" pos_freq="0.050" neg_freq="0.022" pos_to_neg_ratio="2.30"/&gt;
  &lt;nsp_distribution bin_no="3" nsp_lower_bound_incl="0.50" nsp_upper_bound_excl="1.00" pos_freq="0.106" neg_freq="0.032" pos_to_neg_ratio="3.35"/&gt;
  &lt;nsp_distribution bin_no="4" nsp_lower_bound_incl="1.00" nsp_upper_bound_excl="2.00" pos_freq="0.171" neg_freq="0.033" pos_to_neg_ratio="5.19"/&gt;
  &lt;nsp_distribution bin_no="5" nsp_lower_bound_incl="2.00" nsp_upper_bound_excl="5.00" pos_freq="0.257" neg_freq="0.043" pos_to_neg_ratio="5.94"/&gt;
  ...
&lt;/nsp_information&gt;
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="nsp_distribution">nsp_distribution</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">histogram slice of estimated number of sibling peptides
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>bin_no</td>
                    <td>xs:integer</td>
                    <td>required</td>
                    <td>discretized bin number</td>
                </tr>
                <tr>
                    <td>nsp_lower_bound_incl</td>
                    <td>xs:double</td>
                    <td>optional</td>
                    <td>lower bound nsp value (inclusive)</td>
                </tr>
                <tr>
                    <td>nsp_upper_bound_excl</td>
                    <td>xs:string</td>
                    <td>optional</td>
                    <td>upper bound nsp value (exclusive)</td>
                </tr>
                <tr>
                    <td>nsp_lower_bound_excl</td>
                    <td>xs:double</td>
                    <td>optional</td>
                    <td>lower bound nsp value (exclusive)</td>
                </tr>
                <tr>
                    <td>nsp_upper_bound_incl</td>
                    <td>xs:string</td>
                    <td>optional</td>
                    <td>upper bound nsp value (inclusive</td>
                </tr>
                <tr>
                    <td>pos_freq</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>fraction of correct peptides in bin</td>
                </tr>
                <tr>
                    <td>neg_freq</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>fraction of incorrect peptides in bin</td>
                </tr>
                <tr>
                    <td>pos_to_neg_ratio</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>fraction of correct / fraction of incorrect</td>
                </tr>
                <tr>
                    <td>alt_pos_to_neg_ratio</td>
                    <td>xs:double</td>
                    <td>&nbsp;</td>
                    <td>pos_to_neg_ratio of preceding bin (if greater than that of current bin)</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;nsp_distribution bin_no="0" nsp_lower_bound_incl="0.00" nsp_upper_bound_excl="0.10" pos_freq="0.095" neg_freq="0.666" pos_to_neg_ratio="0.14"/&gt;
&lt;nsp_distribution bin_no="1" nsp_lower_bound_incl="0.10" nsp_upper_bound_excl="0.25" pos_freq="0.040" neg_freq="0.154" pos_to_neg_ratio="0.26"/&gt;
&lt;nsp_distribution bin_no="2" nsp_lower_bound_incl="0.25" nsp_upper_bound_excl="0.50" pos_freq="0.050" neg_freq="0.022" pos_to_neg_ratio="2.30"/&gt;
&lt;nsp_distribution bin_no="3" nsp_lower_bound_incl="0.50" nsp_upper_bound_excl="1.00" pos_freq="0.106" neg_freq="0.032" pos_to_neg_ratio="3.35"/&gt;
&lt;nsp_distribution bin_no="4" nsp_lower_bound_incl="1.00" nsp_upper_bound_excl="2.00" pos_freq="0.171" neg_freq="0.033" pos_to_neg_ratio="5.19"/&gt;
&lt;nsp_distribution bin_no="5" nsp_lower_bound_incl="2.00" nsp_upper_bound_excl="5.00" pos_freq="0.257" neg_freq="0.043" pos_to_neg_ratio="5.94"/&gt;
&lt;nsp_distribution bin_no="6" nsp_lower_bound_incl="5.00" nsp_upper_bound_excl="15.00" pos_freq="0.190" neg_freq="0.031" pos_to_neg_ratio="6.12"/&gt;
&lt;nsp_distribution bin_no="7" nsp_lower_bound_incl="15.00" nsp_upper_bound_excl="inf" pos_freq="0.091" neg_freq="0.019" pos_to_neg_ratio="4.73" alt_pos_to_neg_ratio="6.12"/&gt;
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="ni_information">ni_information</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">learned distributions of estimated number of peptide instances ("number of instances")
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>neighboring_bin_smoothing</td>
                    <td>xs:string</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Subelement Name</th>
                    <th>min</th>
                    <th>max</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td><a href="#ni_distribution">ni_distribution</a></td>
                    <td>1</td>
                    <td>unlim</td>
                    <td>histogram slice of estimated number of sibling peptides</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/ni_information.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;ni_information&gt;
&lt;/ni_information&gt;
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="ni_distribution">ni_distribution</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">histogram slice of estimated number of sibling peptides
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>bin_no</td>
                    <td>xs:integer</td>
                    <td>required</td>
                    <td>discretized bin number</td>
                </tr>
                <tr>
                    <td>ni_lower_bound_incl</td>
                    <td>xs:double</td>
                    <td>optional</td>
                    <td>lower bound nsp value (inclusive)</td>
                </tr>
                <tr>
                    <td>ni_upper_bound_excl</td>
                    <td>xs:string</td>
                    <td>optional</td>
                    <td>upper bound nsp value (exclusive)</td>
                </tr>
                <tr>
                    <td>ni_lower_bound_excl</td>
                    <td>xs:double</td>
                    <td>optional</td>
                    <td>lower bound nsp value (exclusive)</td>
                </tr>
                <tr>
                    <td>ni_upper_bound_incl</td>
                    <td>xs:string</td>
                    <td>optional</td>
                    <td>upper bound nsp value (inclusive)</td>
                </tr>
                <tr>
                    <td>pos_freq</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>fraction of correct peptides in bin</td>
                </tr>
                <tr>
                    <td>neg_freq</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>fraction of incorrect peptides in bin</td>
                </tr>
                <tr>
                    <td>pos_to_neg_ratio</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>fraction of correct / fraction of incorrect</td>
                </tr>
                <tr>
                    <td>alt_pos_to_neg_ratio</td>
                    <td>xs:double</td>
                    <td>&nbsp;</td>
                    <td>pos_to_neg_ratio of preceding bin (if greater than that of current bin)</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>none
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="protein_summary_data_filter">protein_summary_data_filter</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">predicted error/sensitivity for filtering at minimum probability
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>min_probability</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>filter setting</td>
                </tr>
                <tr>
                    <td>sensitivity</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>predicted sensitivity (fraction of correct results passing filter)</td>
                </tr>
                <tr>
                    <td>false_positive_error_rate</td>
                    <td>xs:double</td>
                    <td>required</td>
                    <td>predicted error rate (fraction of results passing filter than are incorrect)</td>
                </tr>
                <tr>
                    <td>predicted_num_correct</td>
                    <td>xs:double</td>
                    <td>&nbsp;</td>
                    <td>predicted number of correct results passing filter</td>
                </tr>
                <tr>
                    <td>predicted_num_incorrect</td>
                    <td>xs:double</td>
                    <td>&nbsp;</td>
                    <td>predicted number of incorrect results passing filter</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>&lt;protein_summary_data_filter min_probability="0.00" sensitivity="1.000" false_positive_error_rate="0.780" predicted_num_correct="246" predicted_num_incorrect="869"/&gt;
&lt;protein_summary_data_filter min_probability="0.10" sensitivity="1.000" false_positive_error_rate="0.187" predicted_num_correct="246" predicted_num_incorrect="56"/&gt;
&lt;protein_summary_data_filter min_probability="0.20" sensitivity="1.000" false_positive_error_rate="0.187" predicted_num_correct="246" predicted_num_incorrect="56"/&gt;
&lt;protein_summary_data_filter min_probability="0.30" sensitivity="0.969" false_positive_error_rate="0.119" predicted_num_correct="238" predicted_num_incorrect="32"/&gt;
&lt;protein_summary_data_filter min_probability="0.40" sensitivity="0.945" false_positive_error_rate="0.083" predicted_num_correct="232" predicted_num_incorrect="21"/&gt;
  ...
&lt;protein_summary_data_filter min_probability="1.00" sensitivity="0.350" false_positive_error_rate="0.000" predicted_num_correct="86" predicted_num_incorrect="0"/&gt;
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="intensity">intensity</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>mz</td>
                    <td>xs:float</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>ratio</td>
                    <td>xs:float</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>error</td>
                    <td>xs:float</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>none
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="fragment_masses">fragment_masses</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>channel</td>
                    <td>positiveInt</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>mz</td>
                    <td>xs:float</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>none</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="isotopic_contributions">isotopic_contributions</a>&gt;</h2>

<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">none<br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Subelement Name</th>
                    <th>min</th>
                    <th>max</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td><a href="#contributing_channel">contributing_channel</a></td>
                    <td>1</td>
                    <td>unlim</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/isotopic_contributions.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>none
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="contributing_channel">contributing_channel</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>channel</td>
                    <td>positiveInt</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Subelement Name</th>
                    <th>min</th>
                    <th>max</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td><a href="#affected_channel">affected_channel</a></td>
                    <td>1</td>
                    <td>unlim</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Graphical Context:</b></td>
        <td class="zero"><img width="530" src="img/contributing_channel.png">
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>none
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>

<h2>Element &lt;<a name="affected_channel">affected_channel</a>&gt;</h2>
<table class="zero">
    <tbody>
    <tr>
        <td class="zero"><b>Definition:</b></td>
        <td class="zero">none
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Attributes:</b></td>
        <td class="zero">
            <table border="1">
                <tbody>
                <tr>
                    <th>Attribute Name</th>
                    <th>Data Type</th>
                    <th>Use</th>
                    <th>Definition</th>
                </tr>
                <tr>
                    <td>channel</td>
                    <td>positiveInt</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                <tr>
                    <td>correction</td>
                    <td>xs:float</td>
                    <td>required</td>
                    <td>&nbsp;</td>
                </tr>
                </tbody>
            </table>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Subelements:</b></td>
        <td class="zero">none
            <br>
        </td>
    </tr>
    <tr>
        <td class="zero"><b>Example Context:</b></td>
        <td class="zero"><pre>none
</pre>
        </td>
    </tr>
    </tbody>
</table>
<br>
</body>

</html>
